contents/blog/multi-product-pricing.md

```diff
@@ -0,0 +1,73 @@
+
+We charged by event ingested. Obviously, we wound up with abuse - a handful of high volume users would send zero events, but would use our Javascript snippet to do just session replay, to store _millions_ of recordings.
+
+We decided a multi product pricing strategy had long term advantages, and - to be frank - we had to stop people abusing recording, and we didn't want to limit it for users already paying us a healthy amount.
+
+## It's a way to take extra products seriously
+
+The most important thing, but the one we didn't anticipate, was charging for session recording very quickly led to more usage.
+
+Eh? 
+
+Positioning. If you see two products listed side-by-side on a website, you know each is being treated as a first class citizen. Beforehand, we were "product analytics++". That meant no one was taking our session replay feature seriously.
+
+The moment we started charging, more users realized the feature existed, the more seriously they took it, and the more demanding their feedback. Paying users are less bug tolerant!
+
+We put a small team on it, who quickly iterated with customers and improved the quality.
+
+Growth of session replay took off - it now has similar daily usage to product analytics.
+
+## Focusing on paying customers can improve the free product
+
+Intuitively, if someone is paying, they will have a higher expectation.
+
+However, the secondary effect we didn't anticipate is that fixing the smaller issues can lead to much more usage from both paid and free users. It turns out that in 2021, a lot of UX weirdness and buginess we had was affecting all users of our platform, but our free users weren't complaining that loudly.
+
+As context, PostHog spent the first year and a half prioritizing our open source project - we wanted to build an inbound sales model, so we could double down on product rather than investing in cold calling. Leverage, baby. This meant we'd need a great free product and a reasonably large community. After 18 months, we were ready to focus on our first paid product - product analytics.
+
+Our only company goal at this point was to get 5 reference, paying customers. By going really deep with this handful, we realized that we needed to improve our funnel experience - a core part of our product. We '[nailed funnels](new-vp-nailing-funnels)', and revenue started climbing as a result. However, delightfully, our _free_ user numbers also started growing much faster.
+
+This journey has since repeated itself with session recording. We have a small team of 2 working on this product, and we set them the goal in Q4 last year of charging for their product and getting their first paying customers. Since the free and paid session recording products are the same, except for a usage limit with the free product, focusing on more-demanding, paid users led to similar quality improvements. 
```

Comment:
```suggestion
This journey has since repeated itself with session replay. Since the free and paid session recording products are the same, except for a usage limit with the free product, focusing on more-demanding, paid users led to similar quality improvements for everyone.
```

Reply:
I'm assuming you removed the reference to 2 people and charging revenue because it feels scary to put this out there?

Reply:
No, was more to avoid repetition / contradiction from the previous section where we talk about putting a small team on it + just to shorten the section a bit.